### ISSUE TITLE
[Feature request | documentation]: add documentation on how to open, edit, and save

### ISSUE THREAD

**Issue by weavermonkey** [scanny/python-pptx#717]
The current documentation addresses how to make presentations from scratch. The use case it does not cover in the same level of is how to open an existing presentation, edit values, and save it. There are lots of questions on SO on this topic, and having a wiki page will be of great help

**Comment by MartinPacker** [comment]
When you make a presentation from scratch you are, in effect, opening an existing one and editing it. Even if it is the default one.

Without reviewing the documentation to check, I would guess what you need more of is examples of how to navigate a presentation's elements.

**Comment by weavermonkey** [comment]
@MartinPacker yup - exactly. So what I was hoping for was examples following the same style as existing ones, but focused on editing values inside an already existing template.

tagging @scanny

**Comment by MartinPacker** [comment]
I could imagine a (tree) diagram with Presentation at the top, progressing down through Slides, Slide, Shapes, Shape etc.

Then after that examples of how to get from eg Presentation to Shapes.

(I'm not good at drawing diagrams, BTW, and I don't know what open source tool to use that would be acceptable for this repo.)
